tests/network/vmi_multus.go

```diff
@@ -258,6 +271,30 @@ var _ = Describe(SIG("Multus", Serial, decorators.Multus, func() {
 				By("checking virtual machine instance can ping using ptp cni plugin")
 				Expect(libnet.PingFromVMConsole(detachedVMI, multiIfaceNetGatewayIP2)).To(Succeed())
 			})
+
+			It("VirtualMachineInstance with cni ptp plugin interface with custom MAC address", func() {
```

Comment:
It is ok to include it in a group with the single secondary network test, but it will be used with one NIC.